production/ksonnet/loki/config.libsonnet

```diff
@@ -12,11 +12,12 @@
     // flag for tuning things when boltdb-shipper is current or upcoming index type.
     using_boltdb_shipper: true,
 
-    wal_enabled: false,
+    wal_enabled: true,
     query_scheduler_enabled: false,
     overrides_exporter_enabled: false,
 
     // flags for running ingesters/queriers as a statefulset instead of deployment type.
+    // WAL enabled configurations automatically use statefulsets.
```

Comment:
The jsonnet lib tries to detect if a WAL is being used and if so, use k8s statefulsets instead of deployments and I wanted to leave a note that mentioned this explicitly. The relevant code I'm referring to is here: https://github.com/grafana/loki/blob/main/production/ksonnet/loki/wal.libsonnet#L7

Reply:
Okay. with the line above it's ambiguous if its something that requires stateful queriers (vs just ingesters).

Reply:
The comment _precedes_ the `stateful_ingesters` line, which is what it's actually commenting (I also find the github display for this a bit weird): 
![image](https://user-images.githubusercontent.com/8173478/140067829-63bdacbf-730a-4408-b099-1bf03ad7437a.png)